.github/workflows/ci.yml

```
@@ -13,7 +13,7 @@ jobs:
 
     steps:
       # Checkout the npm/cli repo
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
```

The fetch depth defaults to `1` with the new version of `actions/checkout`. Ordinarily I don't think this would matter, but we've seen some weirdness with `depth: 1` checkouts, and I'd rather avoid this for now. We can always adjust this later in the future.

```suggestion
    - uses: actions/checkout@v2
      with:
        fetch-depth: 0
```

---

From the docs on [`actions/checkout`](https://github.com/actions/checkout#usage)

    # Number of commits to fetch. 0 indicates all history.
    # Default: 1
    fetch-depth: ''